ocpp/v20/schemas/AuthorizeRequest_v1p0.json

```diff
@@ -1,6 +1,5 @@
 ﻿{

```

Comment:
Do you think there is a way we can inform why we have removed this id reference, so next time with even new schemas we know we have to remove it?

Reply:
schade